util/src/main/java/io/grpc/util/GracefulSwitchLoadBalancer.java

```diff
@@ -112,20 +112,6 @@ public GracefulSwitchLoadBalancer(Helper helper) {
     this.helper = checkNotNull(helper, "helper");
   }
 
-  @Override
-  public void handleResolvedAddresses(ResolvedAddresses resolvedAddresses) {
```

Comment:
You can't just delete this. It is needed as long as this method in on the API. Mark it `@Deprecated`; `@Deprecated` functions can all other deprecated functions without warning.

Reply:
Marked as ```@Deprecated```